docs/MIGRATING_V1.md

```diff
@@ -0,0 +1,316 @@
+# Upgrading to Project Factory v1.0
+
+The v1.0 release of Project Factory is a backwards incompatible release and
+features significant changes, specifically with how GSuite resources are
+managed. A state migration script is provided to update your existing states
+to minimize or eliminate resource re-creations. 
+
+NOTE: This document is a work in progress as v1.0 has not yet been released.
+Terraform module sources will be changed when the module is released.
+
+## Migration Instructions
+
+### Without GSuite
+
+This migration was performed with the following example configuration.
+
+```hcl
+/// @file simple_project/main.tf
+
+provider "google" {
+  credentials = "${file(var.credentials_path)}"
+}
+
+provider "gsuite" {
```

Comment:
Is the G Suite provider necessary when the integration is disabled?

Reply:
Yes, the gsuite provider produces errors otherwise. If you'd like to verify this you can check against the example code in this document.